src/exporter/DaemonMetricCollector.cc

```diff
@@ -120,12 +118,14 @@ void DaemonMetricCollector::dump_asok_metrics() {
     json_object counter_dump = boost::json::parse(counter_dump_response).as_object();
     json_object counter_schema = boost::json::parse(counter_schema_response).as_object();
 
-    for (auto &labeled_perf : counter_schema) {
-      std::string labeled_perf_group = {labeled_perf.key().begin(), labeled_perf.key().end()};
-      json_object labeled_perf_group_object = labeled_perf.value().as_object();
-      auto counters = labeled_perf_group_object["counters"].as_object();
-      auto counters_labels = labeled_perf_group_object["labels"].as_object();
-      auto labeled_perf_group_counters = counter_dump[labeled_perf_group].as_object()["counters"].as_object();
+    for (auto &perf_group_item : counter_schema) {
+      std::string perf_group = {perf_group_item.key().begin(),
+                                perf_group_item.key().end()};
+      json_object perf_group_object = perf_group_item.value().as_object();
+      auto counters = perf_group_object["counters"].as_object();
+      auto counters_labels = perf_group_object["labels"].as_object();
+      auto labeled_perf_group_counters =
```

Comment:
Kudos for getting rid of "labeled" in most of the names here!  But this variable still has it for some reason, despite the counters in question not necessarily being labeled.  I'd suggest "counters_values" for symmetry with "counters_labels".

Reply:
Actually, this variable specifies the labels field we need to fetch from the dump command.

Reply:
No, it doesn't.  I think you are confusing it with `counters_labels`.